strings.js

```diff
@@ -0,0 +1,263 @@
+// # String Exercises
+//
+// 1. `console.log` the first character of a string.
+
+let string1 = "This is a string";
+console.log(string1[0]);
+
+// 2. `console.log` the length of a string.
+
+let string2 = "To infinity and beyond";
+console.log(string2.length);
+
+// 3. `console.log` the last character of any string.
+
+let string3 = "With great power comes great responsibility";
+console.log(string3[string3.length - 1]);
+
+// 4. Create a code block that takes a single string variable, and returns a
+// copy of the string with the last letter capitalized. For example:
+//
+// ```js
+// 'jimmy'
+// // => 'jimmY'
+// ```
+
+let name = "hupaul";
+let copy = name.slice(0, 5) + name.slice(name.length - 1).toUpperCase();
+console.log(name, copy);
+
+// 5. Create a drEvil code block that will take a single number variable, and
+// log the '<variableAmount> dollars',
+// your code block should add '(pinky)' at the end of the amount if it's 1
+// million. For example:
+//
+// ```js
+// let amount = 10
+// // => 10 dollars
+// let amount = 1000000
+// // =>  1000000 dollars (pinky)
+// ```
+
+let amount = 1000000
+let drEvil = "(pinky)"
+if (amount === 10) {
+  console.log(amount, "dollars")
+} else if (amount === 1000000) {
+  console.log(amount, "dollars " + drEvil)
+}
+
+// 6. Create a `verbing` code block. It should take a single string variable.
+// If its length is at least 3, it should add 'ing' to its end, unless
+// it already ends in 'ing', in which case it should add 'ly' instead.
+// If the string length is less than 3, it should leave it unchanged.
+// For example:
+//
+// ```js
+// verbing('box')
+// // => 'boxing'
+// verbing('train')
+// // => 'training'
+// verbing('swimming')
+// // =>  'swimmingly'
+// verbing('go')
+// // =>  'go'
+// ```
+
+let singleStr = "box";
+if (singleStr.length < 3) {
+  console.log(singleStr);
+} else if (singleStr.length >= 3 && singleStr.slice(singleStr.length - 3) === "ing") {
+  console.log(singleStr + "ly");
+} else if (singleStr.length >= 3) {
+  console.log(singleStr + "ing");
+}
+
+let singleStr2 = "swimming";
+if (singleStr2.length < 3) {
+  console.log(singleStr2);
+} else if (singleStr2.length >= 3 && singleStr2.slice(singleStr2.length - 3) === "ing") {
+  console.log(singleStr2 + "ly");
+} else if (singleStr2.length >= 3) {
+  console.log(singleStr2 + "ing");
+}
+
+let singleStr3 = "go";
+if (singleStr3.length < 3) {
+  console.log(singleStr3);
+} else if (singleStr3.length >= 3 && singleStr3.slice(singleStr3.length - 3) === "ing") {
+  console.log(singleStr3 + "ly");
+} else if (singleStr3.length >= 3) {
+  console.log(singleStr3 + "ing");
+}
+
+
+// 7. Create a `withoutLast` code block that takes a single string variable,
+// and returns a copy of the string without the last word.
+
+let sentence = "You won't take me alive";
+let withoutLast = sentence.slice(0, sentence.length - 5);
```

Comment:
Would this code still work even if we change the sentence to "JRJ Rules!!!!!!!" ? It is crucial for developers to write codes that can accommodate for different inputs as this instance would be considered hard coding.